tests/yadism/esf/test_conv.py

```diff
@@ -119,7 +119,7 @@ def test_symmetric_conv(self):
     @pytest.mark.eko
     def test_basis_function_void(self):
         xg = np.linspace(0.2, 1.0, 5)  # 0.2, 0.4, 0.6, 0.8, 1.0
-        i = InterpolatorDispatcher(xg, 1, False, False)
+        i = InterpolatorDispatcher(xg, 1, False)
```

Comment:
Apparently it doesn't matter, but still let's keep the old behaviour: so please also change here to linear